lib/graph/Invite.js

```diff
@@ -0,0 +1,36 @@
+// dependencies
+var utils = require("../utils");
+var Vertex = require("./Vertex");
+
+
+/**
+ * Represents a invite vertex
+ *
+ * @constructor
+ * @param {Client} client  The API client object (passed around internally)
+ * @param {String} id      UUID
+ */
+function Invite(client, id) {
+    Vertex.call(this, client, id);
+}
+
+// inheritance
+Vertex.extend(Invite);
+
+// override for Vertex#base
+Invite.prototype.base = "invites";
+
+/**
+ * Opens an invite, creating a submission
+ *
+ * @param invite_id
+ * @param accepting_id
+ * @param {Function} callback
+ */
+Invite.prototype.open = function (invite_id, accepting_id, callback) {
```

Comment:
Should add some tests to https://github.com/treetopllc/noble.js/tree/master/lib/graph